app/javascript/utils/formUtils.js

```diff
@@ -44,9 +44,6 @@ export const maxLengthMap = {
   first_name: 40,
   middle_name: 20,
   last_name: 40,
-  day: 2,
```

Comment:
I think we should actually keep these after all. Sorry for the flip-flop!

Reply:
done!